rfc1869.js

```diff
@@ -24,7 +24,7 @@ const chew_regexp = /\s+([A-Za-z0-9][A-Za-z0-9-]*(?:=[^= \x00-\x1f]+)?)$/;
 
 exports.parse = (type, line, strict) => {
     let params = [];
-    line = (new String(line)).replace(/\s*$/, '');
+    line = (String(line)).replace(/\s*$/, '');
```

Comment:
Why this change? I referred to the docs for String and near as I can tell, this is still the suggested syntax.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/String#return_value

I guess the Sonar folks were thinking of this particular issue of new String objects being usually unnecessary; vs using a String function for desired effects.